backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -80,6 +80,7 @@ private native boolean initialize(String url, String databasePath, String handle
      * Create disabled instance of BacktraceDatabase
      */
     public BacktraceDatabase() {
```

Comment:
Why do we need to have an empty BacktraceDatabase instance if this is disabled by default? Why user (since we're exporting it as a public interface) would like to use it? If this constructor should be available only for unit tests maybe we should make it internal/protected (depends on the unit tests framework capabilities)

Reply:
Unit test requires public functions, this was originally written by @BartoszLitwiniuk

Reply:
I think that we can get rid of this empty constructor but before executing every method on database we need to be sure that database is not null